packages/ember-runtime/lib/mixins/-proxy.js

```diff
@@ -83,9 +79,11 @@ export default Mixin.create({
   */
   content: null,
 
-  _initializeTag: on('init', function() {
+  init() {
+    this._super(...arguments);
+    meta(this).setProxy();
     meta(this)._tag = new ProxyTag(this);
```

Comment:
This has to happen after `init` otherwise you could miss `content` being set inside the user `init`